### PR TITLE
Updated Cordova Quick Start

### PR DESCRIPTION
**Pull request by adrianhall**
- Better global namespace coverage (wraps the code in a closure)
- Includes CSP Meta tag that was missing from original
- Removes 'unsafe-inline' code that is considered bad in Cordova
- Based on QS code from the cordova / taco team

### CODE REVIEW

File: client/cordova/ZUMOAPPNAME/www/js/index.js [hunk 1]
```diff
@@ -16,89 +16,182 @@
...
+
+    /**
+     * Event Handler, called when the host is ready
+     *
+     * @event
+     */
+    function onDeviceReady() {
+        // Device Management - hook up the Pause and Resume events
+        document.addEventListener('pause', function () {
+            console.log('Pausing application');
```
Comment (shrishrirang) [inline client/cordova/ZUMOAPPNAME/www/js/index.js]: console.log will be processed on iOS only while resuming the app. Refer https://cordova.apache.org/docs/en/latest/cordova/events/events.pause.html. It's better to just not have the pause/resume handlers in the quickstart. We don't want to lead the developers down this path only to be surprised later.

File: client/cordova/ZUMOAPPNAME/www/js/index.js [hunk 2]
```diff
@@ -16,89 +16,182 @@
...
-                todoItemTable.insert({ text: itemText, complete: false }).then(refreshTodoItems, handleError);
-            }
-            textbox.val('').focus();
-            evt.preventDefault();
-        });
+        textbox.val('').focus();
+        event.preventDefault();
+    }
 
-        $('#refresh').click(function(evt) {
```
Comment (shrishrirang) [inline client/cordova/ZUMOAPPNAME/www/js/index.js]: Is refresh handler registration missing?

**Comment by shrishrirang** [comment]
The QS looks much better now!

File: client/cordova/ZUMOAPPNAME/www/js/index.js (hunk 2 above)
Comment (adrianhall) [inline client/cordova/ZUMOAPPNAME/www/js/index.js]: Great point - will add it

File: client/cordova/ZUMOAPPNAME/www/js/index.js (hunk 1 above)
Comment (adrianhall) [inline client/cordova/ZUMOAPPNAME/www/js/index.js]: Agreed - I'll comment out the console and add a comment instead.

**Comment by shrishrirang** [comment]
Looks good.

**Comment by adrianhall** [comment]
Want to merge and deploy?

**Pull request merged**